content/features/features.rst

```diff
@@ -42,182 +42,8 @@ To send a Broadcast Message:
-(they must both use the same number of digits) and then the values are
-concatenated.
-
-Two less frequently used systems, the Global Area Reference System
-(`GARS <http://en.wikipedia.org/wiki/Global_Area_Reference_System>`__)
-and Geographic Reference System
-(`GEOREF <http://en.wikipedia.org/wiki/Georef>`__), divide the world
-into polygonal areas based on different sets of nested grids and encode
-locations as a string of grid ID values.
-
-.. figure:: arcgis-1.png
-   :alt: arcgis 1
-
-Each of these ways of representing a location results in different
-strings of characters with a variety of formats.  This table shows the
-coordinate representation for Philadelphia, Pennsylvania in the US (75°
-9’ 18.382” West longitude by 39° 59’ 0.637” North latitude) in each of
-these systems.  This is not an exhaustive list of the delimiters and
-hemisphere indicators that are used in each system.
-
-ArcGIS 10 can convert coordinates expressed in each of these systems
-into the others, and provides ways of displaying and accepting
-coordinate input in most of them.
-
-**Displaying Coordinates**
-
-.. figure:: arcgis-2.png
-   :alt: arcgis 2
-
-You can see the coordinate value for a place on a map in the ArcMap
-status bar, at the bottom right of the application. The default
-coordinate format in the status bar is based on the units of the first
-layer added to the map, often feet, meters, or degrees of
-longitude/latitude.
-
-.. figure:: arcgis-3.png
-   :alt: arcgis 3
-
-You can change the coordinate display to use any of these systems
-(except GARS and GEOREF) by right-clicking Layers in the table of
-contents and selecting Properties. On the General tab, under Units, you
-can select the type of coordinates you would like to see, for example
-MGRS.
-
-**Go To XY Coordinates**
-
-.. figure:: arcgis-4.png
-   :alt: arcgis 4
-
-If you have a set of coordinates and you want to find their location on
-the map, you can use the Go To XY tool.
-
-.. figure:: arcgis-5.png
-   :alt: arcgis 5
-
-Open the tool from the toolbar, then select a coordinate format. Type
-(or paste) the coordinates into the text box and click one of the
-buttons to Pan to, Zoom to, Flash, Add a Point, Add a Point Labeled with
-Coordinates, or Add a Callout.
-
-The Go To XY tool also keeps a record of recent locations, which you can
-return to by clicking the Recent button and selecting the location. You
-can use this to manually convert coordinates from one format to another.
-To do so, select your input coordinate notation, input your coordinates
-and pan to the location. Then select your new output format and select
-the coordinates from the Recent button.
-
-**Find MGRS coordinates**
-
-.. figure:: arcgis-6.png
-   :alt: arcgis 6
-
-If you work with MGRS coordinates, the Find tool has an MGRS Locator
-that works like the Go To XY tool, but provides additional capabilities
-tailored to the MGRS system.
-
-.. figure:: arcgis-7.png
```

Comment:
Did you delete these image files also?

Reply:
I have pushed a new commit deleting the 12 arcgis images